src/components/Repository.js

```diff
@@ -164,11 +167,13 @@ function Repository({match}) {
               <a rel="noreferrer" target="_blank" href={`https://codetriage.com/${nameWithOwner}`}>
                 <Button primary>Set up CodeTriage</Button>
               </a>
-              {isForked ?
-                <a rel="noreferrer" target="_blank" href={`https://github.com/${user.login}/${repoName}`}>
-                  <Button disabled={isForkLoading} data-test="go-to-fork-button">View fork</Button>
-                </a> :
-                <Button disabled={isForkLoading} onClick={forkRepository}><RepoForkedIcon verticalAlign="middle" /> Fork</Button>}
+              { showFork ? (
```

Comment:
@filiptronicek Instead of ternary condition, we can use just use `showFork && (..` and remove null in else part!